src/model.rs

```diff
@@ -79,6 +79,21 @@ pub struct DeactivateResponse {
     pub dbg_message: String,
 }
 
+#[derive(Clone, Deserialize, Debug, Eq, Hash, PartialEq, Serialize)]
+pub struct ComponentLog {
+    pub id: ComponentId,
+
+    pub dedup_number: u64,
+    pub log: Option<String>,
+
+    pub error: Option<String>,
+}
+
+#[derive(Clone, Deserialize, Debug, Eq, Hash, PartialEq, Serialize)]
+pub struct LogResponse {
```

Comment:
What is a LogResponse used for?

Reply:
It's a JSON object returned by the call to `/meta/logs`